README.md

```diff
@@ -46,6 +46,20 @@ Note that `--defs` supports being listed multiple times, to handle multiple diff
 luau-analyze --defs=globalTypes.d.lua --defs=extraTypes.d.lua fileToAnalyse.luau
 ```
 
+Limit flags:
+
+These options affect internal limits on the Luau typechecker.\
+Changing these may permit the typechecker to work on complex code at the cost of performance.\
+We don't have good documentation on how these flags affect behavior.\
+Defaults may be listed by running `luau-analyze-rojo --help`
```

Comment:
The `\` at the end of these lines allows for a newline without starting a new paragraph. You can usually do this with `\` or `  ` (two spaces) at the end of a line in markdown. I chose `\` because `  ` is easily trimmed by accident.